samples/for.go

```diff
@@ -32,4 +32,10 @@ func main() {
 		}
 		fmt.Println(n)
 	}
+
+	var sequence = []string{"Monday", "Tuesday", "Wednesday", "Thursday", "Friday"}
+	//range function can return index and item in a for loop
```

Comment:
Comments should always have a space after the comment markers - see the other comments in this file.